assets/js/components/Dialog.js

```diff
@@ -47,6 +48,7 @@ const Dialog = ( {
 	confirmButton,
 	dependentModules,
 	danger,
+	showSpinner = false,
```

Comment:
Let's name this a bit more abstractly since the spinner is more of an implementation detail, but also because this prop controls more than just the visibility of the spinner.

How about `inProgress` ?

Reply:
I agree and am very happy to change it. However, `showSpinner` was specified in the IB:
> Add a new boolean prop called showSpinner which is false by default.

Just wanted to check there wasn't some specific reason for this requirement before I change it?